examples/jiralert.yml

```diff
@@ -14,6 +14,8 @@ defaults:
   summary: '{{ template "jira.summary" . }}'
   # Go template invocation for generating the description. Optional.
   description: '{{ template "jira.description" . }}'
+  # Enables reopening existing issues. Optional. Default is true.
+  reopen_enabled: true
   # State to transition into when reopening a closed issue. Required.
   reopen_state: "To Do"
```

Comment:
Can we instead perhaps NOT reopen if there is empty string here? 🤔

Reply:
How about setting `wont_fix_resolution: "Done"` instead? That way, any ticket that is already done will not be reopened.

Reply:
Yea, that would do as well 👍🏽 

To be clear:

We propose optional field in this config "wont_fix_resolution" that would specify issue state which makes jiralert to not reopen existing issues anymore. "Done" means it will NOT reopen any, but ideally you have "Won't fix" or something and manually put jira ticket in this state.

Reply:
> We propose optional field in this config "wont_fix_resolution" that would specify issue state which makes jiralert to not reopen existing issues anymore. "Done" means it will NOT reopen any, but ideally you have "Won't fix" or something and manually put jira ticket in this state.

Just to clarify (because issue submitter and I work together), that's not a fit for our pipeline. Incident tickets can be closed as either fixed or wontfix for a ton of different reasons.

How about a `reopen_re_match` regex that reopens tickets matching the value? So that we can use an empty string to represent "don't reopen anything." Or the inverse, so that we can use `(.*)`